docs/source/03_tutorial/08_package_a_project.md

```diff
@@ -35,6 +35,14 @@ For example, having installed project `kedro-spaceflights` and package `kedro_sp
 python -m kedro_spaceflights
 ```
 
+The packaged project can also be run from within a Python file:
+
+```bash
+from kedro_spaceflights__main__ import main
+
+main()
```

Comment:
Actually I realise Ivan already did this in `develop` with the new optional arguments. 

See https://github.com/kedro-org/kedro/pull/1364/files

Reply:
No worries!